nixos/modules/services/x11/window-managers/nimdow.nix

```diff
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
```

Comment:
This module should be added to `nixos/modules/module-list.nix`.

Reply:
It should also be documented as a new NixOS feature in the release notes (`./nixos/doc/manual/release-notes/rl-2305.section.md`).

Reply:
Thanks. I've updated the module commit with these two changes now.